cache/src/main/java/io/envoyproxy/controlplane/cache/CacheStatusInfo.java

```diff
@@ -17,7 +17,9 @@
   private final T nodeGroup;
 
   private final ConcurrentMap<Long, Watch> watches = new ConcurrentHashMap<>();
+  private final ConcurrentMap<Long, DeltaWatch> deltaWatches = new ConcurrentHashMap<>();
```

Comment:
WDYT about making 2 different implementations `DeltaCacheStatusInfo` and `CacheStatusInfo` that implements same methods? We would have `lastWatchRequestTime` for both depends on implementation.